airflow/cli/commands/local_commands/fastapi_api_command.py

```diff
@@ -93,124 +77,35 @@ def fastapi_api(args):
-        run_args += ["--preload"]
-
-        def kill_proc(signum: int, gunicorn_master_proc: psutil.Process | subprocess.Popen) -> NoReturn:
-            log.info("Received signal: %s. Closing gunicorn.", signum)
-            gunicorn_master_proc.terminate()
-            with suppress(TimeoutError):
-                gunicorn_master_proc.wait(timeout=30)
-            if isinstance(gunicorn_master_proc, subprocess.Popen):
-                still_running = gunicorn_master_proc.poll() is not None
-            else:
-                still_running = gunicorn_master_proc.is_running()
-            if still_running:
-                gunicorn_master_proc.kill()
-            sys.exit(0)
-
-        def monitor_gunicorn(gunicorn_master_proc: psutil.Process | subprocess.Popen) -> NoReturn:
-            # Register signal handlers
-            signal.signal(signal.SIGINT, lambda signum, _: kill_proc(signum, gunicorn_master_proc))
-            signal.signal(signal.SIGTERM, lambda signum, _: kill_proc(signum, gunicorn_master_proc))
-
-            # These run forever until SIG{INT, TERM, KILL, ...} signal is sent
-            GunicornMonitor(
-                gunicorn_master_pid=gunicorn_master_proc.pid,
-                num_workers_expected=num_workers,
-                master_timeout=120,
-                worker_refresh_interval=30,
-                worker_refresh_batch_size=1,
-                reload_on_plugin_change=False,
-            ).start()
-
-        def start_and_monitor_gunicorn(args):
-            if args.daemon:
-                subprocess.Popen(run_args, close_fds=True)
-
-                # Reading pid of gunicorn master as it will be different that
-                # the one of process spawned above.
-                gunicorn_master_proc_pid = None
-                while not gunicorn_master_proc_pid:
-                    sleep(0.1)
-                    gunicorn_master_proc_pid = read_pid_from_pidfile(pid_file)
-
-                # Run Gunicorn monitor
-                gunicorn_master_proc = psutil.Process(gunicorn_master_proc_pid)
-                monitor_gunicorn(gunicorn_master_proc)
-            else:
-                with subprocess.Popen(run_args, close_fds=True) as gunicorn_master_proc:
-                    monitor_gunicorn(gunicorn_master_proc)
-
-        if args.daemon:
-            # This makes possible errors get reported before daemonization
-            os.environ["SKIP_DAGS_PARSING"] = "True"
-            create_app(apps)
-            os.environ.pop("SKIP_DAGS_PARSING")
-
-        pid_file_path = Path(pid_file)
-        monitor_pid_file = str(pid_file_path.with_name(f"{pid_file_path.stem}-monitor{pid_file_path.suffix}"))
-        run_command_with_daemon_option(
-            args=args,
-            process_name="fastapi-api",
-            callback=lambda: start_and_monitor_gunicorn(args),
-            should_setup_logging=True,
-            pid_file=monitor_pid_file,
+        setproctitle(f"airflow-fastapi-{args.hostname}:{args.port}")
```

Comment:
To be more consistent with other `setproctitle` usage in airflow codebase.

```suggestion
        setproctitle(f"airflow fastapi-api -- host:{args.hostname} port:{args.port}")
```
![Screenshot 2025-01-21 at 17 39 21](https://github.com/user-attachments/assets/e74a0f29-c76f-450e-a660-68697af1d4e6)